code/modules/clothing/suits/armor.dm

```diff
@@ -234,6 +234,8 @@
 	return
 
 /obj/item/clothing/suit/armor/reactive/on_block(damage, atom/blocked)
+	if (blocked.alwayspass)
+		return TRUE
```

Comment:
Should return 0 to be more in line with the rest
1 (TRUE) - Reactive armor reacted and teleported away
0 (FALSE) - Reactive armor didn't react